Projects/2-Intermediate/Html-Extractor-App.md

```diff
@@ -0,0 +1,45 @@
+# Html Extractor
```

Comment:
Hey, @rodolfocoding 

Please, add the reference for this exercise on the README.md file!!!

Reply:
Change made :)